tests/processes/ProcessingTest.py

```diff
@@ -288,8 +289,10 @@ def test_process_queues(self):
             self.log_printer,
             self.console_printer)
 
-        self.assertEqual(self.queue.get(timeout=0), ([second_local,
-                                                      third_local]))
+        self.assertEqual(self.queue.get(timeout=0), ([first_local,
+                                                      second_local,
+                                                      third_local,
+                                                      fourth_local]))
         self.assertEqual(self.queue.get(timeout=0), ([fourth_local]))
```

Comment:
@ishanSrt Isn't this redundant since you are checking all these `locals` above only?

Reply:
can't remember exactly but since I am popping from the queue, fourth local must be present two times